tests/commands/run_test.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: UTF-8 -*-
+# -*- coding: utf-8 -*-
```

Comment:
please revert this line

UTF-8 is actually marginally faster at parsing than utf-8